### PR TITLE
Add integration test for webhook and fix broken middleware setup. 

### PR DESCRIPTION
**Pull request by andresuribe87**
The test added failed before, and passed after.

### CODE REVIEW

File: config/dev.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ read_timeout = 5000000000
 write_timeout = 5000000000
 shutdown_timeout = 5000000000
 
-log_location = "log"
+log_location = ""
```
Comment (decentralgabe) [inline config/dev.toml:19]: dev is a local build and we have a log folder where this is works and useful - why remove?

Comment (andresuribe87) [reply config/dev.toml:19]: I found it more useful to log to stdout instead of having to tail a new log location each time.

Comment (decentralgabe) [reply config/dev.toml:19]: is it possible to do both with an additional setting instead of removing this?

Comment (decentralgabe) [reply config/dev.toml:19]: it's been valuable to view logs even after the server dies and do comparisons

**Comment by decentralgabe** [comment]
looks like this is failing in the integration test `/home/runner/work/ssi-service/ssi-service/integration/didweb_integration_test.go:50`

we may want to set the logs in the `integration` pkg to `debug` instead of info it's pretty hard to see what's going on

File: config/dev.toml (hunk 1 above)
Comment (decentralgabe) [reply config/dev.toml:19]: this could work https://gin-gonic.com/docs/examples/write-log/

**Review by decentralgabe — APPROVED** [review]

File: config/dev.toml (hunk 1 above)
Comment (andresuribe87) [reply config/dev.toml:19]: Created https://github.com/TBD54566975/ssi-service/issues/462

**Review by nitro-neal — APPROVED** [review]
Great job! I had something similar on my box but I couldn't get it working all the way. SSI service tried to hit my test server and it would never get through.

Awesome job figuring it out!

**Pull request merged**